internal/tiltfile/flags/flags.go

```diff
@@ -88,9 +109,28 @@ func (e *Extension) parse(thread *starlark.Thread, fn *starlark.Builtin, args st
 		return starlark.None, err
 	}
 
-	ret, out, err := settings.argDef.parse(e.cmdLineArgs)
+	err = io.RecordReadFile(thread, settings.FlagsState.ConfigPath)
+	if err != nil {
+		return starlark.None, errors.Wrapf(err, "recording read file %s", settings.FlagsState.ConfigPath)
```

Comment:
if this message is useful, shouldn't it live in RecordReadFile? RecordReadFile already has access to all this information